spec/features/comments/user_writes_a_comment_on_a_file_spec.rb

```diff
@@ -0,0 +1,99 @@
+# As a user
+# With access to a file
+# And I go to a file's show page
+# And I type a message in the comment textarea
+# And I click 'Post Comment'
+# Then I expect the current path to be the file's show page
+# And I expect the comment to be posted on the page
+
+require 'rails_helper'
+
+RSpec.feature 'A user can leave a comment on a file' do
+  context 'when it is her own file' do
+    scenario 'she can comment on it' do
+      user = create(:user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user) and_return(user)
+
+      folder = create(:folder)
+      text = create(:text_binary)
+      folder.binaries << text
+      user.home.folders << folder
+
+      visit "/#{user.username}/home/#{folder.route}"
```

Comment:
Folders now have a #url method you can call to generate their path. So here, it would be `visit folder.url`. Does making a folder factory like this work? I thought we had to get the 'home' folder from a user, and then create folders based off of that one